raster/r.buildvrt/r.buildvrt.html

```diff
@@ -48,6 +50,15 @@ <h3>VRT from a DEM in the North Carolina sample dataset</h3>
 r.buildvrt file=tilelist.csv output=elev_state_50m_vrt
 </pre></div>
 
+<h2>KNOWN ISSUES</h2>
+Users may experience significant performance degradation with virtual rasters build
+with <em>r.buildvrt</em> over GDAL-linked (<em>r.external</em>) raster maps, especially
+with slower file systems with latency like NFS. Performance degradation can also be
+noticable on local file systems, but is usually less severe.
+<p>
+For GDAL-linked raster maps it is thus recommended to build GDAL VRTs, e.g with
+<em>gdalbuildvrt</em>.
```

Comment:
Could you pls make it "https" for GDAL below? Cannot reach it from the web editing interface.

Reply:
See: [223e8e5](https://github.com/OSGeo/grass/pull/4441/commits/223e8e5a10547a091acdde45f741f5c66f01fb11)